README.md

```diff
@@ -59,7 +59,7 @@ contract MyModule is Module {
 - **[Gnosis Safe](https://gnosis-safe.io)**: The most trusted platform for managing digital assets on Ethereum. Zodiac embraces Gnosis Safe as a powerful, extensible and programmable account standard. Gnosis Safe is the reference implementation of the [IAvatar.sol](contracts/interfaces/IAvatar.sol) interface specified in this library. However, all Zodiac tools are framework agnostic, and they can be plugged into any programmable account that implements the IAvatar interface.
 
 #### Modules
-
+- **[Tellor](https://github.com/gnosis/zodiac-module-reality)** (developed by [Tellor](https://tellor.io)): This module allows on-chain executions based on Snapshot proposal results, it uses the Tellor oracle to retrieve the data in a secure and decentralized manner.
```

Comment:
We haven't really talked internally about how these should be sorted, it was pretty arbitrary when we wrote this.
But as we're now adding in more modules, it might make sense for us to think about ordering in some predictable way. Perhaps just alphabetically would make it easier to navigate?

Reply:
ok, sorted them